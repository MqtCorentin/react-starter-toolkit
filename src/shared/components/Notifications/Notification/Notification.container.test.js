import React from 'react';
import { render } from '@testing-library/react';
import NotificationEnhanced from './Notification.container';

describe('NotificationEnhanced', () => {
  it('Render <NotificationEnhanced/>', () => {
    const defaultProps = {
      type: '',
      classModifier: 'mymodifier',
      id: 'id',
      title: 'title',
      onClose: () => {},
    };
    const { asFragment } = render(<NotificationEnhanced {...defaultProps} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
