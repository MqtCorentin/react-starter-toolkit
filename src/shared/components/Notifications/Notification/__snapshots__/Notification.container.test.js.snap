// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotificationEnhanced Render <NotificationEnhanced/> 1`] = `
<DocumentFragment>
  <div
    class="af-alert af-alert--notification af-alert--mymodifier"
  >
    <button
      class="af-alert__icon-close"
    >
      <i
        class="glyphicon glyphicon-close"
      />
    </button>
    <div
      class="af-alert__title"
    >
      <div
        class="af-alert__title-icon"
      >
        <i
          class="glyphicon glyphicon-undefined"
        />
      </div>
      <div
        class="af-alert__title-text"
      >
        title
      </div>
    </div>
  </div>
</DocumentFragment>
`;
